senator/__main__.py

```diff
@@ -1,8 +1,15 @@
 from senator.utility import *
 from senator.game import *
 
-votes, initial_utility = parse_data(load_data(FILE_PATH))
+OWNER_LOSS = 0.59
+OTHER_LOSS = 0.19
+OWNER_TRASHOLD = 0.18 #Set 2.0 to change the model TODO: find better solution than 2.0
```

Comment:
Tyhle věci by nemusely být definované jako konstanty, ale nastavené jako parametry přes `argparse` https://docs.python.org/3/library/argparse.html. Zeptej se na to chatu.

Point je že pak můžeš dělat věci jako `python -m senator --owner_trashold=2`, s tím že když tam nedáš nic tak tam bude nějaký default